internal/service/networkmanager/core_network_policy_model.go

```diff
@@ -64,7 +64,7 @@ type CoreNetworkPolicyCoreNetworkConfiguration struct {
 
 type CoreNetworkEdgeLocation struct {
 	Location         string      `json:"location"`
-	Asn              int         `json:"asn,omitempty"`
+	Asn              string      `json:"asn,omitempty"`
```

Comment:
Will the AWS API still accept the JSON if the type is switched from `int` to `string`?